tests/approaches/test_oracle_approach.py

```diff
@@ -62,6 +63,37 @@ def test_get_gt_nsrts():
         get_gt_nsrts(set(), set())
 
 
+def test_check_nsrt_parameters():
+    """Checks all of the oracle operators for all envs.
+    """
+    def check_nsrt_parameters(nsrts: Set[NSRT]) -> None:
+        for nsrt in nsrts:
+            effects_vars: Set[Variable] = set()
+            precond_vars: Set[Variable] = set()
+            for lifted_atom in nsrt.add_effects:
+                effects_vars |= set(lifted_atom.variables)
+            for lifted_atom in nsrt.delete_effects:
+                effects_vars |= set(lifted_atom.variables)
+            for lifted_atom in nsrt.preconditions:
+                precond_vars |= set(lifted_atom.variables)
+            assert set(nsrt.option_vars).issubset(nsrt.parameters), \
+                f"Option variables is not a subset of parameters in {nsrt}"
+            for var in nsrt.parameters:
```

Comment:
let's add one more test that the parameters are equal to the union of all four things